scripts/esbuild/util.ts

```diff
@@ -0,0 +1,89 @@
+import { join } from 'path';
+
+import { BuildOptions } from '../utils/options';
+
+/**
+ * Aliases which map the module identifiers we set in `paths` in `tsconfig.json` to
+ * bundles (built either with esbuild or with rollup).
+ *
+ * @param opts options for the current build
+ * @returns a map of aliases to bundle entry points, relative to the root directory
+ */
+export function getEsbuildAliases(opts: BuildOptions): Record<string, string> {
+  return {
+    // node module redirection
+    chalk: 'ansi-colors',
+
+    // mapping aliases to top-level bundle entrypoints
```

Comment:
yeah, I think it will need to grow a bit. Right now it covers the ones I needed to add to get the `cli` and then `compiler` bundles building correctly and externalizing each other.